cmd/run_test.go

```diff
@@ -81,3 +85,183 @@ func TestRunWithoutConfig(t *testing.T) {
 		})
 	}
 }
+
+func directoryExists(path string) bool {
+	_, err := os.Stat(path)
+	return !os.IsNotExist(err)
```

Comment:
I'd us afero DirExists for this purpose, since I don't believe stat actually checks that it's a directory - this will also catch other types of errors I believe 

https://github.com/spf13/afero/blob/v1.9.5/util.go#L254

Reply:
Another good callout! Fixed the tests.